oc-pushy-pedant/Gemfile

```diff
@@ -1,8 +1,8 @@
 source "https://rubygems.org"
 
 # Pinned to the same version shipped in the current pushy client
-gem "chef", "= 13.4.24"
-gem "ohai", "= 13.0.1"
+gem "chef", "14.1.12"
+gem "ohai", "~> 14.0"
```

Comment:
we should probably change this comment to something like "These used to be pinned to the same version shipped in the current pushy client, but are currently the latest versions.